src/generic/grid.cpp

```diff
@@ -11337,4 +11400,640 @@ wxGetContentRect(wxSize contentSize,
     return contentRect;
 }
 
+
+#if wxUSE_ACCESSIBILITY
+
+//-----------------------------------------------------------------------------
+// helpers for wxGridAccessible and wxGridCellAccessible
+//-----------------------------------------------------------------------------
+
+// helper to convert row / col into child object id
+// error handling is not required here
+static int GetChildId(wxGrid* grid, int row, int col,
+                      bool isRowHeader, bool isColHeader)
```

Comment:
I [strongly dislike](https://www.wxwidgets.org/develop/coding-guidelines/#no_bool_params) having (especially multiple) boolean parameters in the functions and it looks like we could avoid them her just by treating `row == -1` as column header and `col == -1` as row header, couldn't we?

IMHO this would make the code shorter and more readable.

Reply:
Yes, you're right. This originated from HitTest, where row and col can also be -1 for being outside the cell area, but for the child objects that can't happen.
This now uses m_coords, as suggested below.